packages/map-template/src/components/Switch/Switch.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { useRecoilState, useRecoilValue } from "recoil";
+import primaryColorState from "../../atoms/primaryColorState";
+import './Switch.scss';
+import { ReactComponent as Light2D } from '../../assets/2d-light.svg';
+import { ReactComponent as Dark2D } from '../../assets/2d-dark.svg';
+import { ReactComponent as Light3D } from '../../assets/3d-light.svg';
+import { ReactComponent as Dark3D } from '../../assets/3d-dark.svg';
+import is3DToggledState from "../../atoms/is3DToggledState";
+
+/**
+ * Component responsible for switching between the 2D and 3D features.
+ */
+function Switch() {
```

Comment:
"Switch" is a very generic component name. Can we be more specific?

Reply:
I have brainstormed this quite a lot, and have some options: SwitchViewModes, SwitchMapFeatures, Switch2D3DFeatures. Since the file can't start with a number, 2D3DSwitch is not an option. 
Do you have any preference or any other suggestions? @ammapspeople

Reply:
ViewModeSwitch? VisibilitySwitch?

Reply:
Renamed component to VisibilitySwitch ✅